css/cssom/CSSStyleSheet-constructable-concat.html

```diff
@@ -24,5 +30,5 @@
   assert_equals(shadow.adoptedStyleSheets.length,0);
   shadow.adoptedStyleSheets = shadow.adoptedStyleSheets.concat([sheet]);
   assert_equals(shadow.adoptedStyleSheets.length,1);
-}, "adoptedStyleSheets should allow .concat on empty starting values");
+};
```

Comment:
@mfreed7 You probably want to use `reftest-wait` in order for this to work like you expect? Otherwise it might take a screenshot before the errors are shown.

Reply:
Nothing in the test is async though - all of it should render synchronously as part of window load, so there shouldn’t be a need to do `reftest-wait`, right? This passes on Chromium.

Reply:
Does the error show up in Chromium when you do `assert_equals(false, true)` ? I'm concerned about false passes rather false fails. It's possible that `reftest-wait` is enough, but looking at: https://web-platform-tests.org/writing-tests/reftests.html#controlling-when-comparison-occurs . I don't see anything that mentions explicitly that it waits before all the load event listeners have executed before screenshotting.